neural_lam/train_model.py

```diff
@@ -163,13 +242,31 @@ def main(input_args=None):
         help="Number of example predictions to plot during evaluation "
         "(default: 1)",
     )
+    parser.add_argument(
+        "--save_predictions",
+        action="store_true",
+        help="If predictions should be saved to disk as a zarr dataset "
+        "(default: false)",
+    )
 
     # Logger Settings
     parser.add_argument(
-        "--wandb_project",
+        "--logger",
+        type=str,
+        default="wandb",
```

Comment:
Consider only allowing specified loggers:
```suggestion
        default="wandb",
        choices=["wandb", "mlflow"],
```

Reply:
good point :) 
Done [e1dac04](https://github.com/mllam/neural-lam/pull/77/commits/e1dac044557a850008a13a94fcb4f7f703ed1c3a)